pkg/prober/manager_test.go

```diff
@@ -12,7 +12,7 @@ const (
 	healthyCase   = "healthy case"
 	unhealthyCase = "unhealthy case"
 
-	healthyAddress   = "rancher.com:80"
+	healthyAddress   = "baidu.com:80"
```

Comment:
can we use ip instead, e.g, 8.8.8.8

Reply:
The domain "rancher.com" or IP "8.8.8.8" cannot be visited at home or abroad, but "baidu.com" does.

Reply:
Use 208.67.222.222 instead.